app/uk/gov/hmrc/timetopayproxy/controllers/TimeToPayProxyController.scala

```diff
@@ -38,6 +39,7 @@ class TimeToPayProxyController @Inject() (
   authoriseAction: AuthoriseAction,
   cc: ControllerComponents,
   timeToPayProxyService: TTPQuoteService,
+  @unused
   fs: FeatureSwitch
```

Comment:
Are you sure this cannot be removed? I haven't seen most of the code, so what are the consequences of removing this parameter? Do the tests start complaining? (in which case it might be better to keep it here)